### PR TITLE
Update RELEASE_PROCESS

### PR DESCRIPTION
**Pull request by nicosomb**
Some changes on release process.

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ During this documentation, we assume the release is `$LAST_WALLABAG_RELEASE` (li
 #### Create a new release on GitHub
 
 - [Create the new release on GitHub](https://github.com/wallabag/wallabag/releases/new) by targetting the `master` branch or any appropriate branch (for instance backports).
-- Update [website](https://github.com/wallabag/website) to change the redirect rule for `/latest-v2-package` & `/latest-v2`. They both should redirect to the asset of the GitHub release.
+- Update [website](https://github.com/wallabag/website) to change downloads, MD5 sum, releases, the new blog post and the redirect rule for `/latest-v2-package` & `/latest-v2`. They both should redirect to the asset of the GitHub release.
 - Update Dockerfile https://github.com/wallabag/docker (and create a new tag)
-- Update [website](https://github.com/wallabag/website) website (downloads, MD5 sum, releases and new blog post)
+- [Create the new Docker release on GitHub](https://github.com/wallabag/docker/releases/new).
```
Comment (j0k3r) [inline RELEASE_PROCESS.md:22]: This is already explained line 21

Comment (nicosomb) [reply RELEASE_PROCESS.md:22]: Oops, sorry.

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -17,9 +17,8 @@ During this documentation, we assume the release is `$LAST_WALLABAG_RELEASE` (li
 #### Create a new release on GitHub
 
 - [Create the new release on GitHub](https://github.com/wallabag/wallabag/releases/new) by targetting the `master` branch or any appropriate branch (for instance backports).
-- Update [website](https://github.com/wallabag/website) to change the redirect rule for `/latest-v2-package` & `/latest-v2`. They both should redirect to the asset of the GitHub release.
+- Update [website](https://github.com/wallabag/website) to change downloads, MD5 sum, releases, the new blog post and the redirect rule for `/latest-v2-package` & `/latest-v2`. They both should redirect to the asset of the GitHub release.
```
Comment (j0k3r) [inline RELEASE_PROCESS.md:20]:
```suggestion
- Update [website](https://github.com/wallabag/website) to change MD5 sum and create the release blog post (based on the changelog).
```

**Review by j0k3r — COMMENTED** [review]
Looks great, you can squash :+1:

**Pull request merged**